S21/Akhilraj03-Github/codereview.py

```diff
@@ -0,0 +1,96 @@
+## Main Code for Lower case Conversion for all Languages dynamically
+
+## Defining a Funtion to convert the Upper case letters to Lower case letters with word and langauge variables
+
+def UpperCasetoLowerCaseConverter(word,language):
+	word=word
+	language=language
+		
+		## Lower Case Convertion language of English and other languages extensions of english
+
+		if(language=='english' or language=='en' or language=='en-US' or language=='en-Latn' or language=='en-Latin' or language=='en-IE'):
+			for letter in word:
+				letter=word.lower() 		## Convert Each letter to lower case letter
+			string=letter
+			print(string)				## Printing word output
+			print(string.islower()) 		## Checking whether the coversion for English is Sucessful or not
+		
+		## Lower Case Conversion for Turkish & Azerbaijani languages
+
+		if(language=='turkish' or language=='tr' or language=='azerbaijani' or language=='az'):
+			for letter in word:
+
+				if(letter=='i' or letter=='I'):
+					I=u"\u0131"
+					print(I,end='')
+				else:
+					print(letter.lower(),end='')
+		
+				if(string==letter or string==I)
+					print(string.islower())		
+			
+		## Lower Case Conversion for Greek language
+
+		if(language=='greek' or language=='el')
```

Comment:
similar to turkish, we can add "Greek" here too? what do you think ?

Reply:
done